### DISCUSSION TITLE
Does your plugin detect what version is running and delete and replace it with your one?

### DISCUSSION THREAD

**Discussion by egpd1701** [nguyenquyhy/Flight-Tracker-StreamDeck#317] [General]
Hi

Some weird stuff happening now.

For a little while now, after i compile and deploy my code, as soon as i run it, it gets replaced with your code. 

This is not being copied from another location on my machine as I removed it and only have my version there.

Also, when i build the plugin, the first time it runs, it copies my code to the Plugins directory but after it runs, it gets changed to then copy over your code.

What is going on and how do we stop this happening?

**Comment by rmroc451** [comment]
This was happening to me last year. 8t had something to do with the version number in the streamdeck profile not matching what is out in the streamseck store.

It detects a diff and updates to what is in the store.

I can't remember how I stopped this, maybe turned off auto updates from streamdeck itself?

![Screenshot_20211225-103253_Discord-Beta.jpg](https://user-images.githubusercontent.com/15070945/147389454-0c1c5b7a-ed59-4917-a20d-0af60243a99b.jpg)

Looks like I put in a firewall block

**Reply by nguyenquyhy** [reply]
Just need to update the version to the same as current version in Stream Deck store, right now is 0.39 (manually or by a rebase with the main repository), or change the plugin ID in the manifest. 😂

**Comment by egpd1701** [comment]
Cheers - fixed now.
